lenskit/algorithms/basic.py

```diff
@@ -153,6 +153,51 @@ def inverse_transform(self, ratings):
             rvps['rating'] += rvps['u_off'].fillna(0)
         return rvps.drop(columns=['u_off', 'i_off'])
 
+    def transform_user(self, ratings):
+        """
+        Transform a user's ratings by subtracting the bias model.
+
+        Args:
+            ratings(pandas.Series): The user's ratings, indexed by item.
+                Must have at least `item` as index and `rating` column.
+
+        Returns:
+            pandas.Series:
+                The transformed ratings and the user bias.
+        """
+        ratings = ratings.subtract(self.mean_)
+
+        ioff = self.item_offsets_.reindex(ratings.index, fill_value=0)
```

Comment:
I think we have to deal with the (unusual) case when `item_offsets` is None. I think we can protect these lines with

    if self.item_offsets_ is not None:

and do the same in `inverse_transform_user`.